lighthouse-report/test/proto-test.js

```diff
@@ -5,9 +5,11 @@
  */
 'use strict';
 
-const testUtils = require('../test-utils.js');
-const sampleJson = require('../results/sample_v2.json');
-const preprocessor = require('../../lib/proto-preprocessor.js');
+// TODO: should this be moved to lighthouse-core/test/lib/proto-preprocessor-test.js ?
```

Comment:
see comment

Reply:
my initial reaction was this doesn't belong in `lighthouse-report`, so I would say yes